modules/dnn/src/layers/elementwise_layers.cpp

```diff
@@ -329,6 +329,7 @@ struct ReLUFunctor
     {
         lp.type = "ReLU";
         std::shared_ptr<InferenceEngine::ReLULayer> ieLayer(new InferenceEngine::ReLULayer(lp));
+	ieLayer->negative_slope = 0.1f;
```

Comment:
> in darknet ,the negative_slope is eq 0.1, but this value doesn't pass to inferenceengin.

We cannot forget about the rest of supported frameworks so you need to use `slope` member:
```cpp
ieLayer->negative_slope = slope;
```

Anyway darknet's relu has no negative slope at all:

https://github.com/pjreddie/darknet/blob/8215a8864d4ad07e058acafd75b2c6ff6600b9e8/src/activations.h#L34

May I ask you to provide a source?

Reply:
sorry,it is leakyrelu:
static inline float leaky_activate(float x){return (x>0) ? x : .1*x;}
https://github.com/pjreddie/darknet/blob/8215a8864d4ad07e058acafd75b2c6ff6600b9e8/src/activations.h#L38
as you see, the default value is 0.1

Reply:
@dkurt ,But just use it by default backend, in gpu mode(intel inference engine),it's not set the slope value look:
defalut backend-> line 879
Ptr<ReLULayer> ReLULayer::create(const LayerParams& params)
{
    float negativeSlope = params.get<float>("negative_slope", 0.f);
    Ptr<ReLULayer> l(new ElementWiseLayer<ReLUFunctor>(ReLUFunctor(negativeSlope)));
    l->setParamsFrom(params);
    l->negativeSlope = negativeSlope;

    return l;
}
intel inference engine->line 328
#ifdef HAVE_INF_ENGINE
    InferenceEngine::CNNLayerPtr initInfEngine(InferenceEngine::LayerParams& lp)
    {
        lp.type = "ReLU";
        std::shared_ptr<InferenceEngine::ReLULayer> ieLayer(new InferenceEngine::ReLULayer(lp));
        return ieLayer;
    }
#endif  // HAVE_INF_ENGINE

Reply:
@zuoshaobo, that is why I asked you to replace
```cpp
ieLayer->negative_slope = 0.1f;
```
to 
```
ieLayer->negative_slope = slope;
```
both are correct for darknet networks but not for cases with `negative_slope != 0.1`.

Reply:
@dkurt ok, i had  do it now,3q

Reply:
@zuoshaobo, Thank you! Could you please also add an extra space to align that line with the rest? If it possible, squash all the commits into one.